lstchain/tools/lstchain_create_calibration_file.py

```diff
@@ -164,28 +188,26 @@ def start(self):
                     add_config_metadata(calib_data, self.config)
                     add_global_metadata(calib_data, metadata)
 
-
                 # skip first events which are badly drs4 corrected
                 if not self.simulation and count < self.events_to_skip:
                     continue
 
                 # if pedestal event
+                # use a cut on the charge for MC events if trigger not defined
                 if event.trigger.event_type==EventType.SKY_PEDESTAL or (
                     self.simulation and
                     np.median(np.sum(event.r1.tel[tel_id].waveform[0], axis=1))
-                    < self.processor.minimum_hg_charge_median):
-
+                    < self.mc_max_pedestal_adc):
 
                     new_ped = self.processor.pedestal.calculate_pedestals(event)
 
 
-                # if flat-field event: no calibration  TIB for the moment,
-                # use a cut on the charge for ff events and on std for rejecting Magic Lidar events
+                # if flat-field event
+                # use a cut on the charge for MC events if trigger not defined
                 elif event.trigger.event_type==EventType.FLATFIELD or (
-                        self.simulation and np.median(np.sum(event.r1.tel[tel_id].waveform[0], axis=1))
-                        > self.processor.minimum_hg_charge_median
-                        and np.std(np.sum(event.r1.tel[tel_id].waveform[1], axis=1))
-                        < self.processor.maximum_lg_charge_std):
+                        self.simulation and
```

Comment:
what about the cut to get rid of MAGIC LIDAR events?

Reply:
The tagging is now performed by the event source. The event source should not tag LIDAR events as FF so just filtering for the trigger type should be ok.

Reply:
I don't understand why a cut is needed on simulated events. We have perfect information there, so why is there some additional cut needed?

Reply:
Because simulated FF events can be produced also with the LightEmission code (that is what I used in the past) and the trigger is not set there (but could be is it different now?)

Reply:
I never worked with the light emission code, so you are probably right that this is still needed then. Maybe ask Konrad if there is a possibility to override the trigger type when using this?